Gemfile

```diff
@@ -43,6 +43,7 @@ group :development do
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
   gem 'ruby-prof'
+  gem 'hirb'
```

Comment:
please indicate what hirb does / is for?

Reply:
it allows us to display the sql database in the rails console as a table for easier reading. I used it during the demo. 
This is what it looks like
```>> Tag.last
+-----+-------------------------+-------------+---------------+-----------+-----------+-------+
| id  | created_at              | description | name          | namespace | predicate | value |
+-----+-------------------------+-------------+---------------+-----------+-----------+-------+
| 907 | 2009-03-06 21:10:41 UTC |             | gem:tags=yaml | gem       | tags      | yaml  |
+-----+-------------------------+-------------+---------------+-----------+-----------+-------+
1 row in set```

Reply:
that's cool!  I meant there should be a comment in the Gemfile like `for db table display via rails console` or something.